src/main/java/com/easypost/model/PostageLabel.java

```diff
@@ -39,7 +39,7 @@ public String getId() {
      *
      * @param id ID of this PostageLabel.
      */
-    private void setId(final String id) {
+    public void setId(final String id) {
```

Comment:
Why were these all private to begin with? Does that mean users haven't been able to retrieve any `PostageLabel` properties up until now?

Reply:
It's `setter` only and `getter` is still public in this class previously. `PostageLabel` is the only class that has private `setters` and this needs to be public instead, though most users will probably not use `setter` functions.

Reply:
Thanks for the clarification. Yeah, we're going to explore what properties need to have `setter` functions in the future, but for now, making them public and consistent is good.